packages/plugins/rubygem-smart_proxy_container_gateway/rubygem-smart_proxy_container_gateway.spec

```diff
@@ -1,76 +1,54 @@
-# template: scl
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
-%{!?_root_datadir:%global _root_datadir %{_datadir}}
-
+# template: smart_proxy_plugin
 %global gem_name smart_proxy_container_gateway
 %global plugin_name container_gateway
-%global gem_require_name %{gem_name}
-%global foreman_proxy_dir %{_root_datadir}/foreman-proxy
+
+%global foreman_proxy_min_version 1.24
+%global foreman_proxy_dir %{_datadir}/foreman-proxy
+%global foreman_proxy_statedir %{_localstatedir}/lib/foreman-proxy
```

Comment:
I noticed we can do better here: https://github.com/theforeman/foreman-packaging/pull/8807